ci/scripts/run_ci.sh

```diff
@@ -69,7 +69,7 @@ for pr in ${pr_list}; do
      continue
   fi
   num_cases=0
-  for cases in "${pr_dir}/RUNTESTS/"*; do
+  for cases in "${pr_dir}/RUNTESTS/EXPDIR/"*; do
```

Comment:
These are actually experiments, not `cases`.  Consider `pslot_dir` instead of `cases`.   A `case` is e.g. `C48_S2S`.  A `pslot` is `${case}_${pr_sha}`. A `pslot_dir` is the full path to the `pslot` (as is the case here)

```suggestion
  for pslot_dir in "${pr_dir}/RUNTESTS/EXPDIR/"*; do
```